src/pdfboxing/util.clj

```diff
@@ -0,0 +1,38 @@
+(ns pdfboxing.util)
+
+(defn first-line-valid?
```

Comment:
This seems like a very verbose way to implement an assertion about the first two lines of a file. I think it would be clearer and much shorter to make this a single regular expression.